<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paper</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_open">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_brush">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Brush Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pen Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_drawMode">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Rectangle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_style">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Solid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DashDot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DashDotDot</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_bold">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="specialValueText">
          <string/>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
